FMSgridtools/make_mosaic/regionalgridobj.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import xarray as xr
+import numpy.typing as npt
+
+class RegionalGridObj(self, 
```

Comment:
Maybe the class is not needed here and the write function can be used to regional_mosaic.py